resources/install.rb

```diff
@@ -63,6 +70,15 @@
 property :is_master_ssl_cert, String
 property :is_master_cert_thumbprint, String
 property :is_worker_master_url, String
+property :cf_port, Integer
```

Comment:
I think these ports are covered under the `configure` resource. So the usage is that after the using the install resource to install SQL the configure resource sets ports and encyrption.